### DISCUSSION TITLE
Writing events to CRD

### DISCUSSION THREAD

**Discussion by tomkerkhove** [kubernetes-client/csharp#1398] [Q&A]
Hi,

I was wondering if there are any samples on how to write events to a CRD/RD, for example IngressClass, and if it's even supported?

**Comment by tg123** [comment]
what is `events to a CRD/RD` ? 
you mean event source?

**Reply by tomkerkhove** [reply]
No, write events to a (Custom) Resource Definition

**Reply by tg123** [reply]
could you please explain to me using kubectl
maybe sth i do not know about k8s
